src/pages/ProcessDetail.tsx

```diff
@@ -46,329 +45,252 @@ interface IProps extends RouteComponentProps<MatchParams>, WrappedComponentProps
     query: DecodedValueMap<typeof queryConfig>;
     setQuery: SetQuery<typeof queryConfig>;
 }
+function ProcessDetail({ match, query, setQuery }: IProps) {
+    const actionsRef = useRef(null);
```

Comment:
the ref is not working correctly.

Reply:
which breaks the sticky action button header.

I will have a look at this myself....

Reply:
Fixed in last commit